### PR TITLE
Initialize options for accounts from settings.json

### PR DESCRIPTION
**Pull request by StorytellerCZ**
Noticed this earlier that `Meteor.settings.packages.accounts-base` is not properly initialized on server. I think we used to have this and it might have gotten lost, so I've looked through things and made fixes to ensure those settings get picked up properly.

### CODE REVIEW

File: packages/accounts-base/accounts_common.js [hunk 1]
```diff
@@ -274,15 +282,15 @@ export class AccountsCommon {
     }
 
     // Validate config options keys
-    Object.keys(options).forEach(key => {
+    for (const key of Object.keys(options)) {
```
Comment (denihs) [inline packages/accounts-base/accounts_common.js:285]: Why use `for of` instead of `forEach`?

File: packages/accounts-base/server_main.js [hunk 2]
```diff
@@ -4,9 +4,9 @@ import { AccountsServer } from "./accounts_server.js";
  * @namespace Accounts
  * @summary The namespace for all server-side accounts-related methods.
  */
-Accounts = new AccountsServer(Meteor.server, Meteor.settings.packages?.accounts || {});
+Accounts = new AccountsServer(Meteor.server, Meteor.settings.packages?.['accounts-base'] || {});
 // TODO[FIBERS]: I need TLA
-Accounts.init().then()
+Accounts.init().then();
```
Comment (denihs) [inline packages/accounts-base/server_main.js:9]: Can you check if TLA works here?

File: packages/accounts-base/server_main.js [hunk 3]
```diff
@@ -4,9 +4,9 @@ import { AccountsServer } from "./accounts_server.js";
  * @namespace Accounts
  * @summary The namespace for all server-side accounts-related methods.
  */
-Accounts = new AccountsServer(Meteor.server, Meteor.settings.packages?.accounts || {});
+Accounts = new AccountsServer(Meteor.server, Meteor.settings.packages?.['accounts-base'] || {});
```
Comment (denihs) [inline packages/accounts-base/server_main.js:7]: So, there isn't an `accounts` option that can be provided here? :thinking:

If someone already provides `accounts` as a setting, wouldn't this break their code?

File: packages/accounts-base/accounts_common.js (hunk 1 above)
Comment (StorytellerCZ) [reply packages/accounts-base/accounts_common.js:285]: Better performance according to: https://romgrk.com/posts/optimizing-javascript

File: packages/accounts-base/server_main.js (hunk 3 above)
Comment (StorytellerCZ) [reply packages/accounts-base/server_main.js:7]: There was no documented `accounts`, it was actually `accounts-base`, but I'm fine with using both.

File: packages/accounts-base/server_main.js (hunk 2 above)
Comment (StorytellerCZ) [reply packages/accounts-base/server_main.js:9]: I will, but that is out of scope of this PR.

Comment (StorytellerCZ) [reply packages/accounts-base/server_main.js:9]: Run Meteor from checkout with `await Accounts.init();` and didn't get any error. I can change the code here if you want.

Comment (denihs) [reply packages/accounts-base/server_main.js:9]: We can leave it to another PR

**Review by denihs — APPROVED** [review]

**Pull request merged**